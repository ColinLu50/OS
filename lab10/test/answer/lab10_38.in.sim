25659
1070
547
534
547
534
