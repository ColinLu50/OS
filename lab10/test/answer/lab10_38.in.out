FCFS
8 -> 364 -> 367 -> 389 -> 537 -> 311 -> 45 -> 84 -> 172 -> 379 -> 240 -> 504 -> 334 -> 462 -> 105 -> 524 -> 539 -> 537 -> 246 -> 19 -> 175 -> 464 -> 397 -> 92 -> 507 -> 432 -> 182 -> 381 -> 163 -> 129 -> 314 -> 374 -> 514 -> 417 -> 437 -> 89 -> 319 -> 95 -> 156 -> 157 -> 398 -> 495 -> 241 -> 291 -> 0 -> 468 -> 308 -> 303 -> 261 -> 120 -> 512 -> 526 -> 98 -> 57 -> 13 -> 397 -> 230 -> 421 -> 68 -> 57 -> 423 -> 104 -> 86 -> 484 -> 368 -> 66 -> 272 -> 469 -> 120 -> 384 -> 388 -> 335 -> 484 -> 322 -> 264 -> 507 -> 30 -> 481 -> 222 -> 415 -> 526 -> 150 -> 383 -> 139 -> 451 -> 83 -> 444 -> 351 -> 278 -> 450 -> 242 -> 441 -> 175 -> 456 -> 444 -> 45 -> 250 -> 385 -> 89 -> 258 -> 277 -> 231 -> 538 -> 420 -> 396 -> 330 -> 80 -> 279 -> 181 -> 202 -> 21 -> 300 -> 315 -> 314 -> 282 -> 121 -> 192 -> 307 -> 172 -> 319 -> 274 -> 55 -> 124 -> 509 -> 162 -> 83 -> 274 -> 27 -> 504 -> 291 -> 73 -> 419 -> 507 -> 73 -> 136 -> 394 -> 409 -> 341
25659
SSTF
8 -> 13 -> 19 -> 21 -> 27 -> 30 -> 45 -> 45 -> 55 -> 57 -> 57 -> 66 -> 68 -> 73 -> 73 -> 80 -> 83 -> 83 -> 84 -> 86 -> 89 -> 89 -> 92 -> 95 -> 98 -> 104 -> 105 -> 120 -> 120 -> 121 -> 124 -> 129 -> 136 -> 139 -> 150 -> 156 -> 157 -> 162 -> 163 -> 172 -> 172 -> 175 -> 175 -> 181 -> 182 -> 192 -> 202 -> 222 -> 230 -> 231 -> 240 -> 241 -> 242 -> 246 -> 250 -> 258 -> 261 -> 264 -> 272 -> 274 -> 274 -> 277 -> 278 -> 279 -> 282 -> 291 -> 291 -> 300 -> 303 -> 307 -> 308 -> 311 -> 314 -> 314 -> 315 -> 319 -> 319 -> 322 -> 330 -> 334 -> 335 -> 341 -> 351 -> 364 -> 367 -> 368 -> 374 -> 379 -> 381 -> 383 -> 384 -> 385 -> 388 -> 389 -> 394 -> 396 -> 397 -> 397 -> 398 -> 409 -> 415 -> 417 -> 419 -> 420 -> 421 -> 423 -> 432 -> 437 -> 441 -> 444 -> 444 -> 450 -> 451 -> 456 -> 462 -> 464 -> 468 -> 469 -> 481 -> 484 -> 484 -> 495 -> 504 -> 504 -> 507 -> 507 -> 507 -> 509 -> 512 -> 514 -> 524 -> 526 -> 526 -> 537 -> 537 -> 538 -> 539 -> 0
1070
SCAN
8 -> 0 -> 0 -> 13 -> 19 -> 21 -> 27 -> 30 -> 45 -> 55 -> 57 -> 66 -> 68 -> 73 -> 80 -> 83 -> 84 -> 86 -> 89 -> 92 -> 95 -> 98 -> 104 -> 105 -> 120 -> 121 -> 124 -> 129 -> 136 -> 139 -> 150 -> 156 -> 157 -> 162 -> 163 -> 172 -> 175 -> 181 -> 182 -> 192 -> 202 -> 222 -> 230 -> 231 -> 240 -> 241 -> 242 -> 246 -> 250 -> 258 -> 261 -> 264 -> 272 -> 274 -> 277 -> 278 -> 279 -> 282 -> 291 -> 300 -> 303 -> 307 -> 308 -> 311 -> 314 -> 315 -> 319 -> 322 -> 330 -> 334 -> 335 -> 341 -> 351 -> 364 -> 367 -> 368 -> 374 -> 379 -> 381 -> 383 -> 384 -> 385 -> 388 -> 389 -> 394 -> 396 -> 397 -> 398 -> 409 -> 415 -> 417 -> 419 -> 420 -> 421 -> 423 -> 432 -> 437 -> 441 -> 444 -> 450 -> 451 -> 456 -> 462 -> 464 -> 468 -> 469 -> 481 -> 484 -> 495 -> 504 -> 507 -> 509 -> 512 -> 514 -> 524 -> 526 -> 537 -> 538 -> 539
547
C-SCAN
8 -> 0 -> 0 -> 539 -> 539 -> 538 -> 537 -> 526 -> 524 -> 514 -> 512 -> 509 -> 507 -> 504 -> 495 -> 484 -> 481 -> 469 -> 468 -> 464 -> 462 -> 456 -> 451 -> 450 -> 444 -> 441 -> 437 -> 432 -> 423 -> 421 -> 420 -> 419 -> 417 -> 415 -> 409 -> 398 -> 397 -> 396 -> 394 -> 389 -> 388 -> 385 -> 384 -> 383 -> 381 -> 379 -> 374 -> 368 -> 367 -> 364 -> 351 -> 341 -> 335 -> 334 -> 330 -> 322 -> 319 -> 315 -> 314 -> 311 -> 308 -> 307 -> 303 -> 300 -> 291 -> 282 -> 279 -> 278 -> 277 -> 274 -> 272 -> 264 -> 261 -> 258 -> 250 -> 246 -> 242 -> 241 -> 240 -> 231 -> 230 -> 222 -> 202 -> 192 -> 182 -> 181 -> 175 -> 172 -> 163 -> 162 -> 157 -> 156 -> 150 -> 139 -> 136 -> 129 -> 124 -> 121 -> 120 -> 105 -> 104 -> 98 -> 95 -> 92 -> 89 -> 86 -> 84 -> 83 -> 80 -> 73 -> 68 -> 66 -> 57 -> 55 -> 45 -> 30 -> 27 -> 21 -> 19 -> 13
534
LOOK
8 -> 0 -> 13 -> 19 -> 21 -> 27 -> 30 -> 45 -> 55 -> 57 -> 66 -> 68 -> 73 -> 80 -> 83 -> 84 -> 86 -> 89 -> 92 -> 95 -> 98 -> 104 -> 105 -> 120 -> 121 -> 124 -> 129 -> 136 -> 139 -> 150 -> 156 -> 157 -> 162 -> 163 -> 172 -> 175 -> 181 -> 182 -> 192 -> 202 -> 222 -> 230 -> 231 -> 240 -> 241 -> 242 -> 246 -> 250 -> 258 -> 261 -> 264 -> 272 -> 274 -> 277 -> 278 -> 279 -> 282 -> 291 -> 300 -> 303 -> 307 -> 308 -> 311 -> 314 -> 315 -> 319 -> 322 -> 330 -> 334 -> 335 -> 341 -> 351 -> 364 -> 367 -> 368 -> 374 -> 379 -> 381 -> 383 -> 384 -> 385 -> 388 -> 389 -> 394 -> 396 -> 397 -> 398 -> 409 -> 415 -> 417 -> 419 -> 420 -> 421 -> 423 -> 432 -> 437 -> 441 -> 444 -> 450 -> 451 -> 456 -> 462 -> 464 -> 468 -> 469 -> 481 -> 484 -> 495 -> 504 -> 507 -> 509 -> 512 -> 514 -> 524 -> 526 -> 537 -> 538 -> 539
547
C-LOOK
8 -> 0 -> 539 -> 538 -> 537 -> 526 -> 524 -> 514 -> 512 -> 509 -> 507 -> 504 -> 495 -> 484 -> 481 -> 469 -> 468 -> 464 -> 462 -> 456 -> 451 -> 450 -> 444 -> 441 -> 437 -> 432 -> 423 -> 421 -> 420 -> 419 -> 417 -> 415 -> 409 -> 398 -> 397 -> 396 -> 394 -> 389 -> 388 -> 385 -> 384 -> 383 -> 381 -> 379 -> 374 -> 368 -> 367 -> 364 -> 351 -> 341 -> 335 -> 334 -> 330 -> 322 -> 319 -> 315 -> 314 -> 311 -> 308 -> 307 -> 303 -> 300 -> 291 -> 282 -> 279 -> 278 -> 277 -> 274 -> 272 -> 264 -> 261 -> 258 -> 250 -> 246 -> 242 -> 241 -> 240 -> 231 -> 230 -> 222 -> 202 -> 192 -> 182 -> 181 -> 175 -> 172 -> 163 -> 162 -> 157 -> 156 -> 150 -> 139 -> 136 -> 129 -> 124 -> 121 -> 120 -> 105 -> 104 -> 98 -> 95 -> 92 -> 89 -> 86 -> 84 -> 83 -> 80 -> 73 -> 68 -> 66 -> 57 -> 55 -> 45 -> 30 -> 27 -> 21 -> 19 -> 13
534
